### PR TITLE
DCMP to master - first offseason CR

### PR DESCRIPTION
**Pull request by tal935**

### CODE REVIEW

File: src/main/java/edu/greenblitz/tobyDetermined/Robot.java [hunk 1]
```diff
@@ -33,6 +34,7 @@
 
 public class Robot extends TimedRobot {
 	
+	AnalogInput pressureSwitch;
```
Comment (asaf8888) [inline src/main/java/edu/greenblitz/tobyDetermined/Robot.java:37]: non subsystem switch

File: src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java [hunk 2]
```diff
@@ -92,16 +92,16 @@ public static class Frankenstein {
             public static final double ROBOT_WIDTH_IN_METERS = 0.79;
             public static final double BUMPER_WIDTH = 0.08;
 
-            public static final double MAX_VELOCITY = 4;
-            public static final double MAX_ACCELERATION = 4;
+            public static final double MAX_VELOCITY = 2;
```
Comment (asaf8888) [inline src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java:95]: should be clear this is not actual max but only limiting factor

File: src/main/java/edu/greenblitz/tobyDetermined/commands/Auto/FullCubeHigh.java [hunk 3]
```diff
@@ -4,13 +4,14 @@
...
 import edu.greenblitz.tobyDetermined.commands.telescopicArm.claw.EjectCube;
 import edu.greenblitz.tobyDetermined.commands.telescopicArm.elbow.StayAtCurrentAngle;
+import edu.greenblitz.tobyDetermined.commands.telescopicArm.goToPosition.GoToPosition;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
 
 public class FullCubeHigh extends SequentialCommandGroup {
 	public FullCubeHigh(){
 		super(
-				new PlaceFromAdjacent(RobotMap.TelescopicArm.PresetPositions.CUBE_HIGH).raceWith(new WaitCommand(5)).deadlineWith(new DefaultRotateWhenCube()),
```
Comment (asaf8888) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/Auto/FullCubeHigh.java:13]: place from adjacent still relevent?

Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/commands/Auto/FullCubeHigh.java:13]: yes

**Pull request merged**